extensions/sql-migration/src/constants/strings.ts

```diff
@@ -820,6 +943,7 @@ export const ISSUES_DETAILS = localize('sql.migration.issues.details', "Issue de
 export const SELECT_DB_PROMPT = localize('sql.migration.select.prompt', "Click on SQL Server instance or any of the databases on the left to view its details.");
 export const NO_ISSUES_FOUND_VM = localize('sql.migration.no.issues.vm', "No issues found for migrating to SQL Server on Azure Virtual Machine.");
 export const NO_ISSUES_FOUND_MI = localize('sql.migration.no.issues.mi', "No issues found for migrating to SQL Server on Azure SQL Managed Instance.");
```

Comment:
Even here, I think this should just be "No issues found for migrating to Azure SQL Managed Instance.".

I think SQL VM is the only one whose full product name is "SQL Server on Azure Virtual Machine", the rest are just "Azure SQL Database" / "Azure SQL Managed Instance"